pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -101,9 +105,10 @@ def name(self):
         """The name of the device."""
         return "default.qubit.2"
 
-    def __init__(self, seed=None) -> None:
+    def __init__(self, seed=None, max_workers=None) -> None:
```

Comment:
Please could you also add to the args in the docs [here](https://github.com/PennyLaneAI/pennylane/pull/4319/files#diff-803f2937d933d089284f5a1b3715656b7f2eea59ea09de2eb86381dbeca1a2c0R45)?